### ISSUE TITLE
How to use err.retry() ?

### ISSUE THREAD

**Issue by tracycollins** [totemstech/instagram-node#63]
I think I'd like to use this, but there's no documentation on how it's used.

Does is use a callback function?  Does it take any arguments?

Can someone point me at an example?

Thanks

**Comment by matiaslarsson** [comment]
You can always check the source if the docs are sketchy. For example;
```
user_media_recent = function(user_id, options, cb) {
    var retry = function() {
        user_media_recent(user_id, options, cb);
    };
    ....
```

As you can see it calls itself again with the exact same arguments. It sure could be helpful to have some kind of err.retry iterator that increases after each retry so you could stop it after say three tries though!

**Comment by tracycollins** [comment]
thanks Matias. I did try to figure out what it does from the source, but I'm a JS newbie and wasn't really sure that I'd understood correctly. 

i agree. a bit more control over the retry would be helpful, considering how buggy i'm finding the instagram API to be.

**Comment by matiaslarsson** [comment]
Yeah, one can spend some time trying to figure out other peoples (and your own) code sometimes. ;)

You could create a new issue for having better retry control and hope the maintainer or someone else creates a pull request.
